### DISCUSSION TITLE
Debug Export Button

### DISCUSSION THREAD

**Discussion by TheDog-GH** [triplea-game/triplea#10520] [Feature Requests & Ideas]
@asvitkine 
A request to help me, help you 😉 

I would like an export button for all the Debug> Hard_AI>Show Logs into one text file.
With a button to export the lot with an auto generated filename.
Filename = TripleA + triplea_version + map_name_scenario (=filename of xml) + date (YY-MM-DD) + time (HH:MM:SS)

I spend too much time, looking at the history tab of a game or in the Hard_AI logs ^A ^C paste into Notepad++ and search.

or any variant on the above would be much appreciated. 

It would immediately help with giving feedback on Constructions and consume other units.

**Comment by DanVanAtta** [comment]
@TheDog-GH is this still something you would like?

**Reply by TheDog-GH** [reply]
Yes, but if your time is limited, then I would like the 2.6 Map downloads to be working regularly, more.
As there are lots of new maps that cannot be downloaded within 2.6, especially two of mine
Shogun Advanced
1941 Global Command Decision
Thanks!
Good to have you back! 😁
